boards/arm/rp2040/common/include/rp2040_bmp180.h

```diff
@@ -0,0 +1,84 @@
+/****************************************************************************
+ * boards/arm/rp2040/common/include/rp2040_bmp180.h
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __BOARDS_ARM_RP2040_COMMON_INCLUDE_BOARD_BMP180_H
+#define __BOARDS_ARM_RP2040_COMMON_INCLUDE_BOARD_BMP180_H
```

Comment:
```suggestion
#ifndef __BOARDS_ARM_RP2040_COMMON_INCLUDE_RP2040_BMP180_H
#define __BOARDS_ARM_RP2040_COMMON_INCLUDE_RP2040_BMP180_H
```

Reply:
@gustavonihei , thanks for the review. I've just fixed it. Please, would you review it again?